src/components/Calculator.js

```diff
@@ -1,114 +1,12 @@
-import React, { useState } from 'react';
-import calculate from '../logic/calculate';
+import React from 'react';
+import Button from './Button';
 import '../styles.css';
 
 function Calculator() {
```

Comment:
- [ ] [Optional]👍 You've done a great job by using arrow functions in most of your functional components; they're more concise and aligned with modern JavaScript practices. However, in the `Calculator` component, you've used the old function component. I recommend using arrow functions consistently throughout your project, as they are a more modern and cleaner approach. Keep up the good work! 👏🚀📝